listener/common_test.go

```diff
@@ -6,10 +6,11 @@ import (
 	"app/base/mqueue"
 	"app/base/utils"
 	"fmt"
-	"github.com/RedHatInsights/patchman-clients/inventory"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
```

Comment:
There is no reason to do any changes in imports, please revert it.

Reply:
Interesting this change was created automatically and I could not revert it while I was commiting this PR